lib/takeaway.rb

```diff
@@ -0,0 +1,25 @@
+
+class Takeaway
+	
+	def initialize(list:, order: nil)
+		@list = list
+		@order = order || Order.new
+	end
+
+	def display_list
+		list.display
+	end
+
+
+	def place_order(dishes)
+    dishes.each do |dish, quantity|
+    order.add(dish, quantity)
+  	end
```

Comment:
Lint/BlockAlignment: `end` at 17, 3 is not aligned with `dishes.each do |dish, quantity|` at 15, 4.